### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+FEATURES:
+* **New Resource:** `google_apigee_environment_keyvaluemaps_entries` ([#7717](https://github.com/hashicorp/terraform-provider-google-beta/pull/7717))
+* **New Resource:** `google_apigee_environment_keyvaluemaps` ([#7717](https://github.com/hashicorp/terraform-provider-google-beta/pull/7717))
+* **New Resource:** `google_compute_resize_request` ([#7725](https://github.com/hashicorp/terraform-provider-google-beta/pull/7725))
+* **New Resource:** `google_compute_router_route_policy` ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* **New Resource:** `google_scc_v2_organization_mute_config` ([#7744](https://github.com/hashicorp/terraform-provider-google-beta/pull/7744))
+
+IMPROVEMENTS:
+* alloydb: added `observability_config` field to `google_alloydb_instance` resource ([#7737](https://github.com/hashicorp/terraform-provider-google-beta/pull/7737))
+* bigquery: added `resource_tags` field to `google_bigquery_dataset` resource (ga) ([#7718](https://github.com/hashicorp/terraform-provider-google-beta/pull/7718))
```
Comment (shuyama1) [inline CHANGELOG.md:21]:
```suggestion
```
we can omit ga-only changes from the beta changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+* compute: added `custom_learned_ip_ranges` and `custom_learned_route_priority` fields to `google_compute_router_peer` resource ([#7727](https://github.com/hashicorp/terraform-provider-google-beta/pull/7727))
+* compute: added `export_policies` and `import_policies` fields  to `google_compute_router_peer` resource ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* compute: added `shared_secret` field to `google_compute_public_advertised_prefix` resource ([#7767](https://github.com/hashicorp/terraform-provider-google-beta/pull/7767))
+* compute: added `storage_pool` under `boot_disk.initialize_params` to `google_compute_instance` resource ([#7787](https://github.com/hashicorp/terraform-provider-google-beta/pull/7787))
+* compute: changed `target_service` field on the `google_compute_service_attachment` resource to accept a `ForwardingRule` or `Gateway` URL. ([#7736](https://github.com/hashicorp/terraform-provider-google-beta/pull/7736))
+* container: added documentation for `reservation_affinity` field of `google_container_node_pool` resource ([#7741](https://github.com/hashicorp/terraform-provider-google-beta/pull/7741))
```
Comment (shuyama1) [inline CHANGELOG.md:31]: Is this a documentation only change?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+* container: added field `ray_operator_config` for `resource_container_cluster` ([#7795](https://github.com/hashicorp/terraform-provider-google-beta/pull/7795))
```
Comment (shuyama1) [inline CHANGELOG.md:32]:
```suggestion
* container: added field `ray_operator_config` for `google_container_cluster` ([#7795](https://github.com/hashicorp/terraform-provider-google-beta/pull/7795))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+* datastream: added `merge` and `appendOnly` fields to `google_datastream_stream` resource ([#7726](https://github.com/hashicorp/terraform-provider-google-beta/pull/7726))
```
Comment (shuyama1) [inline CHANGELOG.md:33]:
```suggestion
* datastream: added `merge` and `append_only` fields to `google_datastream_stream` resource ([#7726](https://github.com/hashicorp/terraform-provider-google-beta/pull/7726))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+* datastream: promoted `source_config.sql_server_source_config` and `backfill_all.sql_server_excluded_objects` fields in `google_datastream_stream` resource from beta to GA ([#7729](https://github.com/hashicorp/terraform-provider-google-beta/pull/7729))
+* datastream: promoted `sql_server_profile` field in `google_datastream_connection_profile` resource from beta to GA ([#7729](https://github.com/hashicorp/terraform-provider-google-beta/pull/7729))
```
Comment (shuyama1) [inline CHANGELOG.md:35]:
```suggestion
```
These are ga-only changes

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,54 @@
...
+* dlp: added `cloud_storage_target` field to `google_data_loss_prevention_discovery_config` resource ([#7734](https://github.com/hashicorp/terraform-provider-google-beta/pull/7734))
+* netapp: added `default_from_api = true` attribute to `smb_settings` field of GoogleCloudNetappVolumes ([#7770](https://github.com/hashicorp/terraform-provider-google-beta/pull/7770))
```
Comment (shuyama1) [inline CHANGELOG.md:37]: I think this change is to make the field O+C to fix a permadiff, which seems a bug fix.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,54 @@
...
+* resourcemanager: added `check_if_service_has_usage_on_destroy` field to `google_project_service` resource ([#7745](https://github.com/hashicorp/terraform-provider-google-beta/pull/7745))
+* resourcemanager: added the `member` property to `google_project_service_identity` ([#7708](https://github.com/hashicorp/terraform-provider-google-beta/pull/7708))
+* vmwareengine: added `deletion_delay_hours` field to `google_vmwareengine_private_cloud` resource ([#7710](https://github.com/hashicorp/terraform-provider-google-beta/pull/7710))
+* vmwareengine: support type change from `TIME_LIMITED` to `STANDARD` for multi-node `google_vmwareengine_private_cloud` resource ([#7710](https://github.com/hashicorp/terraform-provider-google-beta/pull/7710))
+* workbench: Changed `instance_owners` field in the examples to not use a service account. ([#7777](https://github.com/hashicorp/terraform-provider-google-beta/pull/7777))
```
Comment (shuyama1) [inline CHANGELOG.md:42]: Is this a documentation only change?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,54 @@
...
+* dlp: added `cloud_storage_target` field to `google_data_loss_prevention_discovery_config` resource ([#7734](https://github.com/hashicorp/terraform-provider-google-beta/pull/7734))
+* netapp: added `default_from_api = true` attribute to `smb_settings` field of GoogleCloudNetappVolumes ([#7770](https://github.com/hashicorp/terraform-provider-google-beta/pull/7770))
+* resourcemanager: added `check_if_service_has_usage_on_destroy` field to `google_project_service` resource ([#7745](https://github.com/hashicorp/terraform-provider-google-beta/pull/7745))
+* resourcemanager: added the `member` property to `google_project_service_identity` ([#7708](https://github.com/hashicorp/terraform-provider-google-beta/pull/7708))
+* vmwareengine: added `deletion_delay_hours` field to `google_vmwareengine_private_cloud` resource ([#7710](https://github.com/hashicorp/terraform-provider-google-beta/pull/7710))
+* vmwareengine: support type change from `TIME_LIMITED` to `STANDARD` for multi-node `google_vmwareengine_private_cloud` resource ([#7710](https://github.com/hashicorp/terraform-provider-google-beta/pull/7710))
```
Comment (shuyama1) [inline CHANGELOG.md:41]:
```suggestion
* vmwareengine: supported type change from `TIME_LIMITED` to `STANDARD` for multi-node `google_vmwareengine_private_cloud` resource ([#7710](https://github.com/hashicorp/terraform-provider-google-beta/pull/7710))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,54 @@
...
+* workbench: Changed `instance_owners` field in the examples to not use a service account. ([#7777](https://github.com/hashicorp/terraform-provider-google-beta/pull/7777))
+* workbench: added `access_configs` to `google_workbench_instance` resource ([#7732](https://github.com/hashicorp/terraform-provider-google-beta/pull/7732))
+
+BUG FIXES:
+* compute: fixed perma-diff for `interconnect_type` being `DEDICATED` in `google_compute_interconnect` resource ([#7750](https://github.com/hashicorp/terraform-provider-google-beta/pull/7750))
+* dialogflowcx: fixed intermittent issues with retrieving resource state soon after creating `google_dialogflow_cx_security_settings` resources ([#7772](https://github.com/hashicorp/terraform-provider-google-beta/pull/7772))
+* firestore: fixed missing import of `field` for `google_firestore_field`. ([#7757](https://github.com/hashicorp/terraform-provider-google-beta/pull/7757))
+* firestore: fixes bug where fields `database`, `collection`, `document_id`, and `field` on resources could not be updated. ([#7791](https://github.com/hashicorp/terraform-provider-google-beta/pull/7791))
```
Comment (shuyama1) [inline CHANGELOG.md:49]: need to add resource names

File: CHANGELOG.md (hunk 2 above)
Comment (SarahFrench) [reply CHANGELOG.md:30]: [Yeah](https://github.com/GoogleCloudPlatform/magic-modules/pull/11161/files) - I'll remove via a suggestion!

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,53 @@
...
+* compute: added `custom_learned_ip_ranges` and `custom_learned_route_priority` fields to `google_compute_router_peer` resource ([#7727](https://github.com/hashicorp/terraform-provider-google-beta/pull/7727))
+* compute: added `export_policies` and `import_policies` fields  to `google_compute_router_peer` resource ([#7748](https://github.com/hashicorp/terraform-provider-google-beta/pull/7748))
+* compute: added `shared_secret` field to `google_compute_public_advertised_prefix` resource ([#7767](https://github.com/hashicorp/terraform-provider-google-beta/pull/7767))
+* compute: added `storage_pool` under `boot_disk.initialize_params` to `google_compute_instance` resource ([#7787](https://github.com/hashicorp/terraform-provider-google-beta/pull/7787))
+* compute: changed `target_service` field on the `google_compute_service_attachment` resource to accept a `ForwardingRule` or `Gateway` URL. ([#7736](https://github.com/hashicorp/terraform-provider-google-beta/pull/7736))
+* container: added documentation for `reservation_affinity` field of `google_container_node_pool` resource ([#7741](https://github.com/hashicorp/terraform-provider-google-beta/pull/7741))
```
Comment (SarahFrench) [inline CHANGELOG.md:30]:
```suggestion
```

**Pull request merged**